.travis.yml

```diff
@@ -15,6 +15,15 @@ language: c
 compiler:
   - gcc
 
+matrix:
+  include: 
+  - name: default
```

Comment:
I would be tempted to name it x86_64, but I don't have a strong preference either way.

Reply:
Thank you. Updated .travis.yaml. s390x build is failing https://travis-ci.org/seccomp/libseccomp/jobs/618689980?utm_medium=notification. `tr: write error: Broken pipe`. Could you please give any pointer on this.

Reply:
Hmmm... is it a Travis CI problem, a test problem on s390x problem, or something else entirely?

Do you have access to an s390x box?  I don't :(.  If so, do the tests run successfully on that box when you invoke them by hand?

Reply:
If it helps, I saw similar problems when I tried enabling the aarch64 builds.  Unfortunately I didn't have the time to debug the problem further.  While I haven't used it, Travis does have some provisions for debugging failed builds: https://docs.travis-ci.com/user/running-build-in-debug-mode